project/cmake/scripts/windows/Macros.cmake

```diff
@@ -4,6 +4,7 @@ function(core_link_library lib wraplib)
   # TODO: core_add_library and core_link_library should be refactored so that we have
   #       decoupled functions for generating the shared libraries. Libs that are not
   #       wrapped should directly be compiled into shared libs.
+  message(STATUS "core_add_library lib=${lib}")
```

Comment:
was that just a left over from testing? if so, please remove. I you want a message, it should be meaningful.